src/Endpoints/ProductphotosEndpoint.php

```diff
@@ -0,0 +1,76 @@
+<?php declare(strict_types=1);
+
+namespace JacobDeKeizer\Ccv\Endpoints;
+
+use JacobDeKeizer\Ccv\Exceptions\CcvShopException;
+use JacobDeKeizer\Ccv\Models\Productphotos as Models;
+use JacobDeKeizer\Ccv\Models\Products\Resource\Productphotos;
+
+class ProductphotosEndpoint extends BaseEndpoint
+{
+    /**
+     * @throws CcvShopException
+     */
+    public function get(int $id): Productphotos
+    {
+        $result = $this->doRequest('GET', 'productphotos/' . $id);
+
+        return Productphotos::fromArray($result);
+    }
+
+    /**
+     * @throws CcvShopException
+     */
+    public function delete(int $id): void
+    {
+        $this->doRequest('DELETE', 'productphotos/' . $id);
+    }
+
+    /**
+     * @throws CcvShopException
+     */
+    public function getAllForProduct(int $productId): Models\Collection\Productphotos
+    {
+        $result = $this->doRequest('GET', 'products/' . $productId . '/productphotos');
+
+        return Models\Collection\Productphotos::fromArray($result);
+    }
+
+    /**
+     * @throws CcvShopException
+     */
+    public function create(int $productId, Models\Productphotos\Post $productphoto, bool $onlyFilledProperties = true): void
+    {
+        $this->doRequest(
```

Comment:
Doesn't the create method return the new model, just like the other pr you made?